sshd/issue_invite.sh

```diff
@@ -67,18 +68,8 @@ cat $TMP/id_rsa.pub >> $HOMEDIR/.ssh/authorized_keys
 
 # Output the actual invite code.
 PUBLIC_IP=`cat /hostname`
-export CLOUD_INSTANCE_DETAILS=$(cat << EOF
-{
-  "host":"$PUBLIC_IP",
-  "user":"$USERNAME",
-  "key":"$ENCODED_KEY"
-}
-EOF
-)
+export CLOUD_INSTANCE_DETAILS="\{\"host\":\"$PUBLIC_IP\",\"user\":\"$USERNAME\",\"key\":\"$ENCODED_KEY\"\}"
```

Comment:
Done - was worried it would throw errors as it used to, but just escaping the quotes is sufficient.